relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -194,19 +308,25 @@ pub fn extract_transaction_metrics(
         timestamp,
         tags: match extract_transaction_status(event) {
             Some(status) => with_tag(&tags, "transaction.status", status),
-            None => tags.clone(),
+            None => tags_with_satisfaction.clone(),
         },
     });
 
     // User
     if let Some(user) = event.user.value() {
         if let Some(user_id) = user.id.as_str() {
+            // TODO: If we have a transaction duration of 0 or no transaction at all, does the user count as satisfied?
```

Comment:
I think transactions of duration 0 should count as satisfied. No transaction shouldn't count towards user satisfaction, that's misleading because we actually don't have data.